gradle.properties

```diff
@@ -2,8 +2,8 @@ org.gradle.parallel=true
 
 spinnakerGradleVersion=8.0.4
 pf4jVersion=3.2.0
-korkVersion=7.63.0
+korkVersion=7.59.0
```

Comment:
I downgrade `kork` because `clouddriver 6.11.1-20201012160101` relies on `kork 7.59.0` just to have things consistent